nbbuild/cluster.properties

```diff
@@ -97,6 +97,10 @@ clusters.config.java.list=\
         nb.cluster.3rdparty,\
         nb.cluster.nb
 
+clusters.config.standard-java.list=\
+        ${clusters.config.java.list},\
+        nb.cluster.apisupport
```

Comment:
nb.cluster.apisupport is required without which import of NB modules were failing with below error:

unrecognized project; missing plug-in?

Reply:
The support for NB modules in IDE is part of `apisupport` cluster. There already is a cluster config for that:
```
clusters.config.basic.list=
```
I suggest to use that one instead of `java` or `standard-java`.

Reply:
had used now basic cluster for building